src/subcommand/runner.rs

```diff
@@ -72,10 +72,20 @@ pub(crate) fn runner(args: RunnerArgs) -> Result<i32, ErrorMessage> {
                     .map_err(|e| format!("Failed to wait for QEMU process: {}", e))?;
                 return Err(ErrorMessage::from("Timed Out"));
             }
-            Some(exit_status) => match config.test_success_exit_code {
-                Some(code) if exit_status.code() == Some(code) => 0,
-                other => other.unwrap_or(1),
-            },
+            Some(exit_status) => {
+                #[cfg(unix)]
+                if exit_status.code().is_none() {
+                    use std::os::unix::process::ExitStatusExt;
+                    if let Some(signal) = exit_status.signal() {
+                        eprintln!("QEMU process was terminated by signal {}", signal);
+                    }
+                }
+                let qemu_exit_code = exit_status.code().ok_or("Failed to read QEMU exit code")?;
```

Comment:
This should not say `Failed to read QEMU exit code` when there was a signal.

Reply:
I'm not sure. The cause for the error still is that there is no exit code, so I think it makes sense to print it.